server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/sql/DropColumnsBuilder.java

```diff
@@ -40,33 +46,45 @@ public DropColumnsBuilder(Dialect dialect, String tableName, String... columns)
     this.columns = columns;
   }
 
-  public String build() {
-    StringBuilder sql = new StringBuilder().append("ALTER TABLE ").append(tableName).append(" ");
+  public List<String> build() {
     switch (dialect.getId()) {
       case PostgreSql.ID:
       case MySql.ID:
-        dropColumns(sql, "DROP COLUMN ");
-        break;
+        StringBuilder sql = new StringBuilder().append("ALTER TABLE ").append(tableName).append(" ");
+        dropColumns(sql, "DROP COLUMN ", columns);
+        return Collections.singletonList(sql.toString());
       case MsSql.ID:
-        sql.append("DROP COLUMN ");
-        dropColumns(sql, "");
-        break;
+        return Collections.singletonList(getMsSQLStatement(columns));
       case Oracle.ID:
-        sql.append("DROP (");
-        dropColumns(sql, "");
-        sql.append(")");
-        break;
+        return Collections.singletonList(getOracleStatement());
+      case H2.ID:
+        return Arrays.stream(columns).map(column -> getMsSQLStatement(column)).collect(Collectors.toList(columns.length));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1612)